document-store/src/main/java/org/hypertrace/core/documentstore/mongo/parser/MongoFilteringExpressionParser.java

```diff
@@ -0,0 +1,22 @@
+package org.hypertrace.core.documentstore.mongo.parser;
+
+import com.mongodb.BasicDBObject;
+import java.util.Map;
+import org.hypertrace.core.documentstore.expression.LogicalExpression;
+import org.hypertrace.core.documentstore.expression.RelationalExpression;
+import org.hypertrace.core.documentstore.parser.IFilteringExpressionParser;
+
+public class MongoFilteringExpressionParser implements IFilteringExpressionParser {
+  private final static String FILTER_CLAUSE = "$match";
+
+  @Override
+  public Object parse(LogicalExpression expression) {
+    return null;
```

Comment:
Have just implemented a sample section. Will implement the rest once the API is accepted.